test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SqlProjects/SqlProjectsServiceTests.cs

```diff
@@ -207,23 +224,40 @@ await service.HandleAddPreDeploymentScriptRequest(new SqlProjectScriptParams()
             }, requestMock.Object);
 
             requestMock.AssertSuccess(nameof(service.HandleAddPreDeploymentScriptRequest));
-            Assert.AreEqual(1, service.Projects[projectUri].PreDeployScripts    .Count, "PreDeployScripts count after re-add");
+            Assert.AreEqual(1, service.Projects[projectUri].PreDeployScripts.Count, "PreDeployScripts count after re-add");
+
+            // Validate moving a pre-deployment object script
+            string movedScriptRelativePath = @"SubPath\\RenamedPreDeploymentScript.sql";
```

Comment:
can the `\\` be replaced with `\` since this string has `@` in front?

Reply:
Not just "can" but "should be"!